.github/pull_request_template.md

```diff
@@ -0,0 +1,19 @@
+This PR...
+
+## Changes
+
+-
+
+## Screenshots
+
+(prefer animated gif)
+
+## Checklist
+
+- [ ] your PR is pointed at `next` (unless it's a new release)
+- [ ] tested locally
+- [ ] installed new dependencies
+- [ ] updated the docs (run `yarn prepack`)
```

Comment:
why `yarn prepack`?

Reply:
I initially added this because `oclif` sets up `yarn prepack` to automatically re-generate and update your README with the new commands. So I thought, let's tell people to do that with each PR, so the docs are most up-to-date. But now that we're at this stage, and I've learned that that is probably not the thing we want to do. Removing.

Reply:
ahh, got it. now that I understand what it's for we could just reword if you want to keep. "(run `yarn prepack` if you added a command)

Reply:
Well, it may cause unwanted side effects so let's leave it out actually